packages/utils/src/lib/transform.ts

```diff
@@ -113,12 +117,29 @@ export function toUnixNewlines(text: string): string {
   return platform() === 'win32' ? text.replace(/\r\n/g, '\n') : text;
 }
 
+export function fromJsonLines<T = unknown>(text: string) {
+  const unifiedNewLines = toUnixNewlines(text).trim();
+  return JSON.parse(`[${unifiedNewLines.split('\n').join(',')}]`) as T;
+}
+
+export function toJsonLines<T extends object>(objects: T[]) {
+  return objects.map(obj => JSON.stringify(obj)).join('\n');
+}
+
 export function capitalize<T extends string>(text: T): Capitalize<T> {
   return `${text.charAt(0).toLocaleUpperCase()}${text.slice(
     1,
   )}` as Capitalize<T>;
 }
 
+export function apostrophize(text: string, upperCase?: boolean) {
+  const lastCharMatch = text.match(/(\w)\W*$/);
+  const lastChar =
+    lastCharMatch != null && lastCharMatch[1] ? lastCharMatch[1] : '';
```

Comment:
```suggestion
  const lastChar = lastCharMatch?.[1] ?? '';
```

Reply:
Has that always been a thing? Seems like combining `.` access and indexing 😅  Nice, thank you.

Reply:
Yes. The optional chaining operator has a special syntax for index access of nullish arrays - `arr?.[i]` is shorthand for `arr == null ? undefined : arr[i]`. There's also a similar syntax for safely calling nullish functions - `fn?.(...args)` instead of `fn == null ? undefined : fn(...args)`.

Reply:
Nice!